.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,15 @@
+name: golangci-lint
+on:
```

Comment:
super-nit: vertical white space style is different between the two action config files in this PR, consider picking one style and adopting across both configs?

Reply:
Done.